display_list/testing/dl_rendering_unittests.cc

```diff
@@ -16,6 +16,9 @@
 #include "flutter/fml/math.h"
 #include "flutter/testing/display_list_testing.h"
 #include "flutter/testing/testing.h"
+#ifdef IMPELLER_SUPPORTS_RENDERING
+#include "impeller/typographer/backends/skia/text_frame_skia.h"  // nogncheck
```

Comment:
I think the test likely needs a dependency on this in the gn file for this to compiler correctly

Reply:
Doesn't the nogncheck comment negate that issue? I didn't write it that way, I copied it from ... ?performance overlay?

I didn't see any gn failures on Mac or Windows.

Reply:
I did forget to ifdef the code that used it, though...

Reply:
nogncheck will only silence gn analysis warnings, so it can be used if the dependency is transitive through some other target but if its not compiled in at all then you'll still get the linking errors.

Reply:
it compiles fine on Mac...

Reply:
I removed the comment and fixed the gn error...

Reply:
Try adding "//flutter/impeller/typographer/backends/skia:typographer_skia_backend" to the display_list_rendertests gn target deps.